packages/cli/src/models/config/ConfigManager.ts

```diff
@@ -23,12 +23,21 @@ const ConfigManager = {
 
     const { provider, artifactDefaults } = await this.config.loadNetworkConfig(network, root);
 
-    ZWeb3.initialize(provider);
     Contracts.setSyncTimeout(timeout * 1000);
     Contracts.setArtifactsDefaults(artifactDefaults);
 
-    const txParams = { from: ZWeb3.toChecksumAddress(from || artifactDefaults.from || await ZWeb3.defaultAccount()) };
-    return { network: await ZWeb3.getNetworkName(), txParams };
+    try {
+      ZWeb3.initialize(provider);
+      const txParams = { from: ZWeb3.toChecksumAddress(from || artifactDefaults.from || await ZWeb3.defaultAccount()) };
+      return { network: await ZWeb3.getNetworkName(), txParams };
+    } catch(error) {
+      if (this.config.constructor.name === 'ZosConfig') {
```

Comment:
Is it possible that this gets called before `setBaseConfig`? If so, could `this.config` be null?

Reply:
It shouldn't. `this.setBaseConfig` will always be called before this `if` statement, as it calls `this.initStaticConfiguration` which calls `this.setBaseConfig`.

Reply:
Still, could we handle it just so we don't risk failing in a `catch` block?